community/RAM.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: BlueMap and RAM
+parent: Community Guides
+nav_order: 16
+---
+
+# BlueMap and RAM
+{: .no_toc }
+
+Do not be alarmed if you notice that BlueMap seems to be using a lot of RAM.
+This is actually perfectly normal and should not be an actual problem.
+
+There are multiple things that contribute to this effect:
+
+### 1. Unused RAM
+BlueMap is using **unused** heap-space (RAM) to cache some things
+like e.g. chunk-data and resources to improve performance.
+But as soon as your JVM (your server) needs this cache-space somewhere else, it can do so!
+BlueMap is not hard-reserving that space, so it can't cause an OOM because of this.
+*(Technically this is done using a [SoftValue-Cache](<https://www.baeldung.com/java-caching-caffeine#3-reference-based-eviction>))*  
+After all, it is always said that unused RAM is wasted RAM.
+
+### 2. Garbage Collection
+RAM/Heap management in your JVM is done using a Garbage Collector.
+Your JVM runs this GC in intervals to free up heap-space that is no-longer used.
+If you have a big heap of multiple GBs, depending on your GC-configuration,
+it can wait for a long time until it decides to run the GC and free some space.
+This has the effect that your RAM-Usage will keep climbing until the GC is ran.
+**Or:** until you restart the server.
+
+So if you remove BlueMap to prove that it is using a lot of RAM,
```

Comment:
Not entirely happy with this sentence yet. It is very long. Could use some splitting up.